docs/getting-started/github-apps/github-apps-troubleshooting-tips.rst

```diff
@@ -165,20 +175,29 @@ A new separate workflow/tool will be registered if the original name isn't inclu
 How can I convert my entire existing workflow/tool at once?
 -------------------------------------------------------------
 Currently you cannot convert all existing branches/versions at once. You must add a .dockstore.yml to each branch in order for the GitHub app
-automatically detect and sync changes with the corresponding version on Dockstore.
+automatically detect and sync changes with the corresponding version on Dockstore. A side effect of this is that, unless you edit tagged commits to include .dockstore.yml files, old releases created prior to you creating a .dockstore.yml will not be selectable in Dockstore.
 
 If you have a .dockstore.yml file in your master or develop branches on GitHub, any new branches you create from these as your template
-will have a  .dockstore.yml.
+will have a .dockstore.yml, as will future releases based on master or develop.
 
 :ref:`(back to top) <topGHAFAQ>`
 
 
 Why are only some branches appearing on my workflow/tool?
 ----------------------------------------------------------
-The Dockstore GitHub App is currently unable to parse branches that use special characters besides numerical digits, non-leading dashes, forward slashes, periods, and underscores. "Special characters" includes alphabetical characters with accents, tildes, circumflexes, umlauts, or non-English letters such as ß and ø. These limitations are stricter than what GitHub itself allows. As a result, if you have a GitHub branch named something like `Ó-Fearghail`, `branch-with-{curly-braces}`, or `Robert');-DROP-TABLE-Students;`, that branch will not appear on Dockstore. If you check the Dockstore GitHub App logs, you'll see these branches throw an error such as `Reference refs/heads/branch-with-{curly-braces} is not of the valid form`. 
+Any branches that do not have a .dockstore.yml file will not appear on your workflow/tool entry. If the branches you expect to see do indeed have properly formatted .dockstore.yml file, the culprit might be the name of the branches: The Dockstore GitHub App currently supports only the following special characters in branch names:
+* numerical digits
+* non-leading dashes
+* forward slashes
+* periods
+* underscores.
+
+Branches which contain other special characters, including alphabetical characters with accents, tildes, circumflexes, umlauts, or non-English letters such as ß and ø, cannot be parsed by the Dockstore GitHub App and will not show up. If you check the Dockstore GitHub App logs, you'll see these branches throw an error such as ``Reference refs/heads/Robert');-DROP-TABLE-Students; is not of the valid form``. 
```

Comment:
:raised_eyebrow:

Reply:
I think this whole section can be deleted. See dockstore/dockstore#5354